internals/overlord/checkstate/manager.go

```diff
@@ -387,6 +433,69 @@ type checker interface {
 	check(ctx context.Context) error
 }
 
+func (c *checkData) writeMetrics(writer metrics.Writer) error {
+	checkUp := int64(0)
+	if c.status == CheckStatusUp {
+		checkUp = 1
+	}
+	err := writer.Write(metrics.Metric{
+		Name:       "pebble_check_up",
+		Type:       metrics.TypeGaugeInt,
+		ValueInt64: checkUp,
```

Comment:
I don't really care which way we do this (int64 here or above), but let's at least be consistent between checks and services (we're currently doing it int64-on-variable here and int64-on-field in services). Now that I see them I like the services way better, so let's change this to int64-on-field (this line).